.github/workflows/builds_and_tests.yml

```diff
@@ -134,6 +134,21 @@ jobs:
           name: libwasmvm
           path: ./api/libwasmvm.x86_64.so
 
+  build_static_lib:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up
+        uses: actions/setup-go@v2
```

Comment:
The latest version of `setup-go` is 3.2.1. Please check it.

Reply:
fixed